framework/src/application/node/node.js

```diff
@@ -318,14 +318,18 @@ module.exports = class Node {
 			}),
 			getLastBlock: async () => this.processor.serialize(this.chain.lastBlock),
 			getBlocksFromId: async action =>
-				this.transport.handleRPCGetBlocksFromId(
-					action.params.data,
-					action.params.peerId,
+				this.processor.serialize(
+					this.transport.handleRPCGetBlocksFromId(
```

Comment:
inside this function we have called serialise `return blocks && blocks.map(block => this.chainModule.serialize(block));`
so i dont know if this is correct

Reply:
Oh yeah this is not relevant, I thought it followed the same pattern as `handleRPCGetGetHighestCommonBlock`